dbtmetabase/dbt.py

```diff
@@ -36,12 +36,23 @@ def read_models(self, includes=[], excludes=[]) -> list:
         mb_models = []
 
         for path in (Path(self.project_path) / "models").rglob("*.yml"):
+            logging.info(path)
             with open(path, "r") as stream:
                 schema = yaml.safe_load(stream)
+                if schema is None:
+                    logging.warn(f"SKIPPING EMPTY/INVALID YML: {path}")
+                    continue
                 for model in schema.get("models", []):
-                    name = model["name"]
+                    name = model.get("identifier", model["name"])
+                    print("MODEL", name)
```

Comment:
Either wrap in proper logging or remove.

Reply:
Wrapped in proper logging